apps/perfdash/OWNERS

```diff
@@ -1,4 +1,12 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
 approvers:
-  - bartsmykla
-  - mm4tt
-  - wojtek-t
+- mm4tt
+- wojtek-t
+
+emeritus_approvers:
+- bartsmykla
```

Comment:
AFAIK we haven't seen much approval from this contributor over the last while so it's time to emeritus